Client.php

```diff
@@ -78,21 +77,21 @@ class Client extends Connection {
     /**
      * Create a Websocket client.
      *
-     * @access  public
      * @param   \Hoa\Socket\Client  $client      Client.
      * @param   string              $endPoint    End-point.
      * @param   \Hoa\Http\Response  $request     Response parser.
      * @return  void
      * @throw   \Hoa\Socket\Exception
      */
-    public function __construct ( Socket\Client $client, $endPoint = '/',
-                                  Http\Response $response = null ) {
-
+    public function __construct(Socket\Client $client, $endPoint = '/',
+                                Http\Response $response = null)
+    {
```

Comment:
Which phpcs fixer are you using ?

The one from sensiolabs or the PHP_CodeSniffer

*Hari K T*

You can ring me : +91 9388 75 8821

http://harikt.com , https://github.com/harikt ,
http://www.linkedin.com/in/harikt , http://www.xing.com/profile/Hari_KT

Skype  : kthari85
Twitter : harikt

On Mon, Apr 6, 2015 at 1:44 PM, Ivan Enderlin <notifications@github.com>
wrote:

> In Client.php
> <https://github.com/hoaproject/Websocket/pull/34#discussion_r27788759>:
>
> >       * @param   \Hoa\Socket\Client  $client      Client.
> >       * @param   string              $endPoint    End-point.
> >       * @param   \Hoa\Http\Response  $request     Response parser.
> >       * @return  void
> >       * @throw   \Hoa\Socket\Exception
> >       */
> > -    public function __construct ( Socket\Client $client, $endPoint = '/',
> > -                                  Http\Response $response = null ) {
> > -
> > +    public function __construct(Socket\Client $client, $endPoint = '/',
> > +                                Http\Response $response = null)
> > +    {
>
> Yup, I don't know why php-cs-fixer does not correct this. I have to dig a
> little bit. That's my current goal.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hoaproject/Websocket/pull/34/files#r27788759>.
>